ui/src/importer/importerTaskDetails/EitemImportDetails.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import _isEmpty from 'lodash/isEmpty';
+import { Link } from 'react-router-dom';
+import { Button, Header, List, Modal } from 'semantic-ui-react';
+import { BackOfficeRoutes } from '@inveniosoftware/react-invenio-app-ils';
+
+export class EitemImportDetails extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { open: false };
+  }
+  render() {
+    const { eitemReport } = this.props;
+    const { open } = this.state;
+    return (
+      <Modal
+        onClose={() => this.setState({ open: false })}
+        onOpen={() => this.setState({ open: true })}
+        open={open}
+        trigger={
+          <Button
+            color="red"
+            icon="exclamation"
+            floated="right"
+            size="mini"
+            basic
+          />
+        }
+      >
+        <Modal.Header>EItem - attention required</Modal.Header>
```

Comment:
Why the text `attention required`?

Reply:
because it displays deleted or to be deleted eitems. I thought it might catch some more attention for this case.